build-system/tasks/extension-generator/bento/index.js

```diff
@@ -26,6 +26,7 @@ const fs = require('fs').promises;
 const log = require('fancy-log');
 const path = require('path');
 const {cyan, green, red, yellow} = require('ansi-colors');
+const {insertExtensionBundlesConfig} = require('../');
```

Comment:
Is there any risk here of mutually recursive dependencies? We could consider splitting this method into its own lil' module